chains/solana/contracts/programs/fee-quoter/src/lib.rs

```diff
@@ -115,6 +115,21 @@ pub mod fee_quoter {
             .set_default_code_version(ctx, code_version)
     }
 
+    /// Sets the max_fee_juels_per_msg, which is an upper bound on how much can be billed for any message.
```

Comment:
Could be good to clarify here that juels are different from solana's smallest token denomination for LINK, to prevent misunderstandings on the value to use.